### PR TITLE
feat: add drivable area visualizer (#779)

### PR DESCRIPTION
**Pull request by wep21**
* add drivable area visualizer

* add license

* modify pointed out in pre-commit

* modify pointed out in pre-commit

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [commit-guidelines][commit-guidelines]
- [ ] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by wep21** [comment]
rebase after https://github.com/autowarefoundation/autoware.universe/pull/195 is merged.

File: common/tier4_planning_rviz_plugin/plugins/plugin_description.xml [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
     base_class_type="rviz_common::Display">
-    <description>Display autoware_auto_planning_msgs::Trajectory</description>
+    <description>Display autoware_planning_msg::Trajectory</description>
   </class>
   <class name="rviz_plugins/TrajectoryFootprint"
     type="rviz_plugins::AutowareTrajectoryFootprintDisplay"
     base_class_type="rviz_common::Display">
-    <description>Display autoware_auto_planning_msgs::TrajectoryFootprint</description>
+    <description>Display autoware_planning_msg::TrajectoryFootprint</description>
   </class>
```
Comment (tkimura4) [inline common/tier4_planning_rviz_plugin/plugins/plugin_description.xml:26]: autoware_planning_msg??

Comment (wep21) [reply common/tier4_planning_rviz_plugin/plugins/plugin_description.xml]: fixed at https://github.com/autowarefoundation/autoware.universe/pull/193/commits/362e5172966d85c83d5e97d66b8890c4ae94cffe.

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**